vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -15245,7 +15295,7 @@ unit:OZ_VOL_UK-PER-DAY
   qudt:iec61360Code "0112/2///62720#UAA432" ;
   qudt:uneceCommonCode "J95" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
-  rdfs:label "OZ_UK PER DAY" ;
+  rdfs:label "OZ_VOL_UK PER DAY" ;
```

Comment:
The label doesn't match the qname.

Reply:
All of the OZ_VOL_UK qnames match their labels. What unit are you referring to? I do not see it here.